sites/partners/src/pages/listings/[id]/applications/pending/index.tsx

```diff
@@ -114,10 +116,13 @@ const ApplicationsList = () => {
     formatLinkCell: isListingOpen ? formatDisabledCell : formatEnabledCell,
   }
 
-  const dayjsDate = dayjs(listingDto?.afsLastRunAt)
-  const afsLastRun = {
-    date: dayjsDate.utc().format("MM/DD/YY"),
-    time: dayjsDate.utc().format("hh:mma"),
```

Comment:
I noticed the timestamp was being displayed in UTC time which is confusing since we aren't displaying the timezone. I removed that, but i could be convinced to put this back

Reply:
Seems like this is coming from design-matching discussed [here](https://github.com/bloom-housing/bloom/pull/3168). I'd vote to show it in the user's local time to be consistent with some of the time formatting work [here](https://github.com/bloom-housing/bloom/pull/3526/files)